rust/worker/src/server.rs

```diff
@@ -20,6 +20,7 @@ use chroma_types::chroma_proto::{
     GetVectorsRequest, GetVectorsResponse, QueryVectorsRequest, QueryVectorsResponse,
 };
 use chroma_types::{MetadataValue, ScalarEncoding};
+use kube::core::request;
```

Comment:
nit: unused import?

Reply:
Not sure how it got introduced in the first place. I've removed it.